lib/uplinkc/testdata/helpers.h

```diff
@@ -69,4 +73,37 @@ void with_test_project(void (*handleProject)(ProjectRef)) {
     }
 
     requiref(internal_UniverseIsEmpty(), "universe is not empty\n");
+}
+
+char *mkrndstr(size_t length) { // const size_t length, supra
```

Comment:
this should return arbitrary bytes, and give the buffer as input rather than allocating inside:

```
void mkrndata(buffer *uint8_t, size_t length) {
     for(size_t i = 0; i < length; i++) {
          buffer[i] = uint8_t(i*31);
     } 
}
```

Reply:
done